### ISSUE TITLE
"show chance text" missing from menus? (and rightclick menu issue)

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1152]
Hi,

either I have a typical monday brainwash or "Chance Text" somehow 
isn't in the top menus anymore?
_(I am pretty sure that there was a general "show chance text" in the menus
to set chance for all voices.)_
 
Also right-clicking on the track (waveform) gives chaotic results with the popup window.
Maybe it was there? But I can't access the system fx text column because of the bugged out
popup window.

Best,
tL.

![menuarkward](https://user-images.githubusercontent.com/18437561/45321358-6f54cf00-b545-11e8-8ca6-f631a04d7ff0.gif)

**Comment by kmatheussen** [comment]
"Chance text" hasn't disappeared for me. Isn't it between "Cent text" and
"Velocity text".

I don't know what has happened to the automation popup menu. I'll have to
try it on OSX.
Looks like another Qt bug. I'm definitely going to make custom menus soon,
the Qt ones are really bad.
Is it usable though? Maybe navigating the menus with the keyboard makes it
easier?

**Comment by Lykdraft** [comment]
No, the system subfolder isn't accessible via keyboard shortcuts. It jut doesn't open when getting there.
Also look at the .gif. It seems Chance is really gone here, I don't see cent either...

![menuarkward2](https://user-images.githubusercontent.com/18437561/45358541-05cad400-b5ca-11e8-8d73-3d00a193e5bd.gif)

**Comment by kmatheussen** [comment]
Oh, I thought you meant the track header popup menu. (right-click the track
header)
Cents and change is so uncommon that I probably haven't thought of putting
those into the main menus. It's quite crowded there.

**Comment by Lykdraft** [comment]
Ok, I see. So probably I was searching in the top menu because I can't see or access them in the right-click menu right now.

**Comment by kmatheussen** [comment]
But this is a different right-click menu. The one you above with an error
is the automation popup-menu, this is the track header popup-menu.

**Comment by kmatheussen** [comment]
There's probably a good chance you can fix the automation popup menu by
setting
a lower value under preferences -> Windows -> Maximum number of visible
menu elements



On Tue, Sep 11, 2018 at 4:09 PM, Kjetil Matheussen <k.s.matheussen@gmail.com
> wrote:

> But this is a different right-click menu. The one you above with an error
> is the automation popup-menu, this is the track header popup-menu.
>
>

**Comment by Lykdraft** [comment]
Doesn't work. Can lower value to single digits. Still not working, same results.

**Comment by Lykdraft** [comment]
Oh, it does work kinda on a setting of 10.
But the windows are still all over the place.

**Comment by Lykdraft** [comment]
Also, you're right about the different right-click menus. The one I was looking for is actually the 
right click window on the track header. This works as expected... Aeh, now what?
So the original problem is solved, but we have the right-click menu from the track itself messed up.
What you wanna do? :)

**Comment by kmatheussen** [comment]
Are you sure you can't navigate it with keyboards? Try to to press left
arrow right
after opening the automation menu. Then the first sub menu, which might not
be
visible for you, should close.

**Comment by kmatheussen** [comment]
I've tried on a mac now, and I see the same thing. The main popup menu window size is reduced by half after opening so that half of the entries are not available, and the sub menus pops up at strange positions.

This started to happen after upgrading Qt from 5.7 to 5.10. It's clearly a Qt bug since it doesn't behave like this on the other platforms, but I worked around it by changing the menu into asynchronous mode.

Until next release, I guess you just have to use the "Add automation to current editor track" options for those entries that can't be selected from the popup menu.
